packages/tinacms/src/styles.css

```diff
@@ -1,16 +1,16 @@
-.tina-tailwind {
-  @tailwind base;
-  @tailwind components;
-  @tailwind utilities;
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
 
-  /* gray-600 from tailwind config */
-  color: #565165;
-
-  @layer utilities {
-    .icon-parent {
-      svg {
-        fill: currentColor;
-      }
+@layer utilities {
+  .icon-parent {
+    svg {
+      fill: currentColor;
```

Comment:
The `tailwindcss/nesting` plugin should be the solution for the warning related to `nesting` here:
https://tailwindcss.com/docs/using-with-preprocessors#nesting